packages/babel-plugin-transform-runtime/test/fixtures/runtime-corejs3/es6-for-of/output.js

```diff
@@ -1,4 +1,4 @@
-var _createForOfIteratorHelper = require("@babel/runtime-corejs3/helpers/createForOfIteratorHelper");
+var _createForOfIteratorHelper = require("@babel/runtime-corejs3/helpers/createForOfIteratorHelper").default;
```

Comment:
Here the output changed and it's incompatible with old `@babel/runtime` versions, but it's because we are specifying `"version": "7.100.0"` in the test options.